data/org.gnome.shell.gschema.xml.in

```diff
@@ -158,6 +158,11 @@
       <summary>Pulse Hack desktop icon</summary>
       <description>If true, the Hack desktop icon will pulse to attract the user.</description>
     </key>
+    <key type="b" name="show-hack-launcher">
```

Comment:
I think we should avoid putting any more Endless-specific GSettings in the `org.gnome.shell` namespace, though I accept it's unlikely that upstream will use `show-hack-launcher`.